<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-5I3A2SA0" name="com.pratian.SauceDemo.Automation.Tests.TestAddCartitem$TestAddFlight" tests="1" failures="0" timestamp="2021-02-11T18:44:15 IST" time="0.000" errors="1">
  <testcase name="AddtoCart" time="0.000" classname="com.pratian.SauceDemo.Automation.Tests.TestAddCartitem$TestAddFlight">
    <error type="org.testng.TestNGException" message="
Method public void com.pratian.SauceDemo.Automation.Tests.TestAddCartitem$TestAddFlight.AddtoCart(java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : signIn">
      <![CDATA[org.testng.TestNGException: 
Method public void com.pratian.SauceDemo.Automation.Tests.TestAddCartitem$TestAddFlight.AddtoCart(java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : signIn
at org.testng.internal.Parameters.findDataProvider(Parameters.java:563)
at org.testng.internal.Parameters.handleParameters(Parameters.java:776)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- AddtoCart -->
  <system-out/>
</testsuite> <!-- com.pratian.SauceDemo.Automation.Tests.TestAddCartitem$TestAddFlight -->
